Handwriting/HandwritingIntentCSSValue.md

```diff
@@ -113,21 +113,21 @@ Since the proposed property should not interact with other HTML or IDL attribute
 
 There are no known security impacts of the features in this specification.
 
-## Alternative Solutions
+# Alternative Solutions
 
-The proposal is for this to be an CSS property.
 ## Why not an HTML+IDL attribute?
 
-The [first proposal](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/Handwriting/explainer.md) was to add the handwriting functionality as an HTML+IDL attribute, but after some discussion it was decided that the better option was to implement the functionality in the `touch-action` CSS attribute. [ [1](https://groups.google.com/a/chromium.org/g/blink-dev/c/0r_tV6k0NyA?pli=1)]] [[2](https://github.com/w3c/pointerevents/issues/516)]
+The [first proposal](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/Handwriting/explainer.md) was to add the handwriting functionality as an HTML+IDL attribute which would allow authors to specify whether an element should permit handwriting by adding a new `handwriting(= true|= false|<blank>)` (`<blank>` implying `= true`) attribute to HTML elements.
+
+After some discussion [[1](https://groups.google.com/a/chromium.org/g/blink-dev/c/0r_tV6k0NyA?pli=1)]] [[2](https://github.com/w3c/pointerevents/issues/516)], it became apparent that implementing the functionality in the `touch-action` CSS attribute was the better alternative. The discussion is still ongoing, and we are open to considering different alternatives.
```

Comment:
It would be helpful to enumerate the specific concerns mentioned here, so readers can easily find them without having to search through the entire discussion.